contents/docs/integrate/client/js/index.mdx

```diff
@@ -36,19 +36,41 @@ import JSCaptureIgnoreElements from './snippets/capture-ignore-elements.mdx'
 
 <JSAutoCapture />
 
+#### Tuning autocapture
+
+You can configure autocapture using allowlists. If an array is passed for an allowlist, autocapture events will only be sent for elements matching at least one of the items in the array.
+
+For example, to only capture clicks on buttons on the docs section of the website that contain the data attribute `ph-autocapture` you can do:
+
+```js
+posthog.init('<ph_project_api_key>',
+    {
+        api_host: '<ph_instance_address>',
+        autocapture: {
+                event_allowlist: ['click'], // DOM events from this list ['click', 'change', 'submit']
+                url_allowlist: ['*/docs/*'], // strings or RegExps
```

Comment:
Does this support glob matching or only with regexes? If it's only regexes then this should probably be `.*\/docs\/.*` or perhaps just `\/docs\/` if the regex doesn't need to match the whole url?

Reply:
Improved!